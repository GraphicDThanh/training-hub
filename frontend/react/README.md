# React Redux Training
[Plan doc](https://docs.google.com/document/d/1lYywN0RIhrwsyFZ8uPZuaN5QM78mYuXFD608vS8YEbM/edit?pli=1#)
## ES6
## React Basic
### Practice 1 - examples React
## React Advanced
### Practice 2
## Redux Basic
## Redux Advanced
## Redux Recipes
