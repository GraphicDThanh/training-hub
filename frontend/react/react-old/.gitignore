node_modules
out
dist
.DS_Store
.git
.docpad.db
vendor/
.vagrant
bower-components/
bower_components/
npm-debug.log
.gitignore~
vagrant/Vagrantfile~
build/
