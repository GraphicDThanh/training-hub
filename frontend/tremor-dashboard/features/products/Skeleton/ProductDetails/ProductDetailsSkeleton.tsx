const ProductDetailsSkeleton = () => (
  <div className="flex flex-col md:flex-row w-full lg:w-[30%] h-[400px] rounded-lg animate-pulse gap-2 justify-between mb-10">
    <div className="w-full h-[310px] lg:h-[350px]">
      <div className="m-auto w-[75%] h-full bg-gray-200 dark:bg-gray-700 rounded-lg"></div>
    </div>
  </div>
);

export default ProductDetailsSkeleton;
