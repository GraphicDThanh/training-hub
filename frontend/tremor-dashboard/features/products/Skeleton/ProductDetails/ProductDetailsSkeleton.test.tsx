import { render } from "@testing-library/react";

// Components
import ProductDetailsSkeleton from "./ProductDetailsSkeleton";

describe("ProductDetailsSkeleton component", () => {
  it("should match snapshot", () => {
    const { container } = render(<ProductDetailsSkeleton />);

    expect(container).toMatchSnapshot();
  });
});
