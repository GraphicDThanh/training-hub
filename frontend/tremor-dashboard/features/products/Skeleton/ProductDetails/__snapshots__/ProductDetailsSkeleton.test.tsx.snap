// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductDetailsSkeleton component should match snapshot 1`] = `
<div>
  <div
    class="flex flex-col md:flex-row w-full lg:w-[30%] h-[400px] rounded-lg animate-pulse gap-2 justify-between mb-10"
  >
    <div
      class="w-full h-[310px] lg:h-[350px]"
    >
      <div
        class="m-auto w-[75%] h-full bg-gray-200 dark:bg-gray-700 rounded-lg"
      />
    </div>
  </div>
</div>
`;
